src/main.py

```diff
@@ -69,12 +69,15 @@ def main(module_name: OracleModule):
     kac = KeysAPIClientModule(variables.KEYS_API_URI, web3)
 
     web3.attach_modules({
-        'lido_contracts': LidoContracts,
         'lido_validators': LidoValidatorsProvider,
         'transaction': TransactionUtils,
         'cc': lambda: cc,  # type: ignore[dict-item]
         'kac': lambda: kac,  # type: ignore[dict-item]
     })
+    if variables.LIDO_LOCATOR_ADDRESS:
+        web3.attach_modules({
```

Comment:
there could be some exception if no LIDO_LOCATOR_ADDRESS were setup and we run Accounting module.
Maybe create another entrypoint for this one?